assets/js/settings/blocks/constants.js

```diff
@@ -64,6 +64,8 @@ export const SHIPPING_METHODS_EXIST = getSetting(
 	false
 );
 
+export const PAYMENT_GATEWAY_ORDER = getSetting( 'paymentGatewayOrder', [] );
```

Comment:
Can we avoid using "order"? It confuses with "orders". We should try using "sorting" where possible because it's clearer of intent.

Reply:
I went with `paymentGatewaySortOrder`. Totally unfortunate that we have the clash with `order`!

`paymentGatewaySorting` didn't seem quite right to me, since it's not a sort type (e.g. alpha, etc), it's a manual reordering. 

If we do move this into the config then maybe we could represent as `weight` or `displayPriority` param on each gateway data blob, and we wouldn't have this naming issue :)